src/JsonSchema/PhpBuilder.php

```diff
@@ -176,6 +176,10 @@ private function makeClass($schema, $path)
                 $phpProperty->addMeta($property, self::SCHEMA);
                 $phpProperty->addMeta($name, self::PROPERTY_NAME);
 
+                if (!is_null($property->default)) {
```

Comment:
Let's make this configurable, rendering default can be unwanted in some cases, e.g. to avoid default values when encoding json.

We can have some option like this in `PhpBuilder`:
```php
    /**
     * Use default values to initialize properties
     * @var bool
     */
    public $declarePropertyDefaults = false;
```
and then
```php
                if (!is_null($property->default) && $this->declarePropertyDefaults) {
```

Reply:
Good point